packages/react-vapor/src/components/radio/RadioSelectSelectors.ts

```diff
@@ -13,7 +13,12 @@ const getValue = createSelector(
     (radioSelect: IRadioSelectState) => radioSelect && radioSelect.value
 );
 
+const getDisabledValue = createSelector(

```

Comment:
Unit tests for this new selector?

Reply:
- [ ] Unit Test for getDisabledValue Selector